interceptors-opentelemetry/src/worker/schedule-workflow.ts

```diff
@@ -0,0 +1,30 @@
+import { Connection, WorkflowClient } from '@temporalio/client';
```

Comment:
This file should probably be renamed to `exec-workflow.ts` or whatever we decide to use as a convention.